presto-main/src/main/java/com/facebook/presto/operator/GroupByHash.java

```diff
@@ -64,4 +64,6 @@ static GroupByHash createGroupByHash(
     boolean contains(int position, Page page, int[] hashChannels);
 
     int putIfAbsent(int position, Page page);
+
+    int compare(int leftGroupId, int rightGroupId);
```

Comment:
I don't think this API makes sense. `GroupByHash` is a hash table, so there's no guarantee that groups are comparable. I realize there will be a performance penalty, but I think for now it would be better to write the data into a `Page` using `appendValuesTo` and then sort the `Page`